src/Protocol/Version2.php

```diff
@@ -318,7 +325,7 @@ public static function aeadDecrypt(
         string $header,
         SymmetricKey $key,
         string $footer = ''
-    ): string {
+    ) {

```

Comment:
The extension function may return false too:

https://github.com/jedisct1/libsodium-php/blob/28d13bf437cb969a0583031fc7ac54c5a8dc8116/libsodium.c#L3068

Reply:
If that happens, then we need to be stricter.

Reply:
Isn't https://github.com/paragonie/paseto/pull/113/files#diff-06d71742cf83f525040bc36171832f85796debb414dbae04c7c1132254381ec5R167 enough?